Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Block_IP.yml

```diff
@@ -0,0 +1,733 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: 40612048-e6cb-4448-8774-ea6dadce93ab
+    type: regular
+    task:
+      id: 40612048-e6cb-4448-8774-ea6dadce93ab
+      version: -1
+      name: Push Config
+      description: Push the candidate configuration.
+      script: '|||prisma-sase-candidate-config-push'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      folders:
+        simple: Remote Networks, Mobile Users, Service Connections
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -260,
+          "y": 1505
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 24827071-3736-40b3-8b12-13776b0fe3e1
+    type: condition
+    task:
+      id: 24827071-3736-40b3-8b12-13776b0fe3e1
+      version: -1
+      name: Commit and push configuration to folders?
+      description: Commit and push configuration to folders?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      Auto Commit:
+      - "10"
+      Manual:
+      - "15"
+    separatecontext: false
+    conditions:
+    - label: Auto Commit
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: inputs.AutoCommit
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    - label: Manual
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: inputs.AutoCommit
+            iscontext: true
+          right:
+            value:
+              simple: "False"
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -260,
+          "y": 1140
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: e8535616-fa92-4784-8196-8633d038b74e
+    type: playbook
+    task:
+      id: e8535616-fa92-4784-8196-8633d038b74e
+      version: -1
+      name: Prisma SASE - Add IPs to Static Address Group
+      description: This playbook appends a Static Address Group with provided IPs.
+      playbookName: Prisma SASE - Add IPs to Static Address Group
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "11"
+    scriptarguments:
+      AddressGroupName:
+        complex:
+          root: inputs.StaticAddressGroupName
+      AutoCommit:
+        simple: "False"
```

Comment:
should be from the playbook input

Reply:
It was intended - so that if the user wants to make multiple changes without saving he can do so ( that's why there is another section for commit afterward).